### PR TITLE
Refactor checkbox component styling

### PR DESCRIPTION
**Pull request by ssb-cgn**

### CODE REVIEW

File: src/components/Checkbox/index.jsx [hunk 1]
```diff
@@ -1,24 +1,105 @@
...
+			<label htmlFor={value}>
+				{children}
+				<input
+					id={value}
+					value={value}
+					type="checkbox"
+					checked={selected}
+					onChange={() => callback(value)}
+				/>
+				<span className="check-mark" />
```
Comment (kristofferlund) [inline src/components/Checkbox/index.jsx]: Should be turned into a styled.checkmark with no class needed

File: src/components/Checkbox/index.jsx [hunk 2]
```diff
@@ -1,24 +1,105 @@
...
-				value={value}
-				type="checkbox"
-				checked={selected}
-				onChange={() => callback(value)}
-			/>
-			<span className="check-mark" />
-		</label>
-	</div>
+	<CheckboxWrapper>
+		<div tabIndex={index + 1} className={`checkbox-button-wrapper${disabled ? ' disabled' : ''}`}>
```
Comment (kristofferlund) [inline src/components/Checkbox/index.jsx]: Don't think a div within a div is necessary here, but I could be wrong?

File: src/components/Checkbox/index.jsx [hunk 3]
```diff
@@ -1,24 +1,105 @@
...
+
+    &:after {
+      content: '';
+      display: none;
+      position: absolute;
+
+      left: 6px;
+      top: 2px;
+      width: 6px;
+      height: 12px;
```
Comment (kristofferlund) [inline src/components/Checkbox/index.jsx]: Please fix property order

File: src/components/Checkbox/index.jsx (hunk 1 above)
Comment (ssb-cgn) [reply src/components/Checkbox/index.jsx]: Tror det er bedre nå :)

File: src/components/Checkbox/index.jsx (hunk 2 above)
Comment (ssb-cgn) [reply src/components/Checkbox/index.jsx]: Fixed

File: src/components/Checkbox/index.jsx (hunk 3 above)
Comment (ssb-cgn) [reply src/components/Checkbox/index.jsx]: fixed

**Pull request merged**